resources.html

```diff
@@ -311,9 +295,92 @@ <h6 style="margin-top: 18px; padding-right: 6px;">OLDER</h6>
         </div>
     </section>
     <hr class="hairline">
-    
-     <!-- Loopback integration section -->
+
+    <!-- Presentation videos section -->
     <section class="even-section">
+        <div class="container">
+            <div class="col-lg-12 text-center">
+                <h3 class="section-heading" style="padding-top: 48px;">Presentations</h3>
+                <p>See videos and slides from events.</p>
+            </div>
+            <div class="col-md-12 row">
+                <div class="col-md-6">
+                    <p style="height:6px;"></p>
+                    <div class="text-center" style="justify-content: center"> 
+                        <div class="card-2 pres multiple-links" onclick="location.href='https://www.youtube.com/watch?v=i3UW5PhDoQg';">
```

Comment:
This part is kinda hacky because it won't let you put more than one link in a card or else it'll break up the card.